### ISSUE TITLE
Remove axes

### ISSUE THREAD

**Issue by AliciaSchep** [jonocarroll/ggeasy#6]
Shortcut for 

```r
theme(axis.text.x = element_blank(),
           axis.title.x = element_blank(),
           axis.ticks.x = element_blank(), 
           axis.line.x = element_blank())
```
or same for y.

This could either be two separate functions, `easy_remove_x_axis` and `easy_remove_y_axis`, or one function `easy_remove_axis` with an argument that takes in either "x","y" or "both"

**Comment by jonocarroll** [comment]
At the risk of polluting namespaces (which I'm happy to do in this case) I think all three would be good. I'm all for having as many autocompleting helpers as possible, even if they are just aliases. As many easy ways to do things as possible; one by name, one with a variable selection.

**Comment by AliciaSchep** [comment]
the more the merrier :smile:  

Perhaps the third should be `easy_remove_axes`  (plural) with the default to remove "both" but the option to specify "x" or "y" instead, so `easy_remove_axes("y")` would be same as `easy_remove_y_axis`.

**Comment by jonocarroll** [comment]
I like it! I take it you mean we build the most general one then build a helper on top of that, i.e.

```
easy_remove_y_axis <- function() easy_remove_axes("y")
```

**Comment by AliciaSchep** [comment]
I had been thinking the other way around (the axes one wraps the other two), but your way probably makes more sense!

**Comment by jonocarroll** [comment]
Closed due to b80fe08
